core/src/commonMain/kotlin/com/saveourtool/processbuilder/ProcessBuilderConfig.kt

```diff
@@ -35,11 +39,45 @@ data class ProcessBuilderConfig(
          */
         object Pipe : Redirect
 
+        /**
+         * Output should be discarded
+         */
+        object Null : Redirect {
+            override fun printToFileIfNeeded(stringsToPrint: List<String>): List<String> = emptyList()
+        }
+
+        /**
+         * @param stringsToPrint strings that should be printed
+         * @return [List] of [String] that should be a part of ExecutionResult class
+         */
+        fun printToFileIfNeeded(stringsToPrint: List<String>): List<String> = stringsToPrint
```

Comment:
why you need to return something here?

Reply:
It is done for convenience: 
```Kotlin
ExecutionResult(
    childExitStatus.code,
    stdoutChannel.toList().let(stdout::printToFileIfNeeded),
    stderrChannel.toList().let(stderr::printToFileIfNeeded),
)
```

It may be rewritten with `also` though, but in this case some implementations (e.g. one for `ProcessBuilderConfig.Redirect.Pipe`) will return `Unit`.

Reply:
To my mind both variants are acceptable

Reply:
it's not clear why `printToFileIfNeeded()` returns `emptyList()` in `File`?

Reply:
It was done in order to mock redirects to file: if you redirect to file, you do not expect output to be read through pipe (and be present in `ExecutionResult`)

But it seems there is no need in manual flush, will fix it

Reply:
Now only if `isNullAlias` is true, an emptyList is returned